vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/registry/impl/FSInstallState.java

```diff
@@ -302,7 +302,7 @@ private static WorkspaceFilter readWorkspaceFilter(Element child) throws Configu
      */
     public void save(Path file) throws IOException {
         Files.createDirectories(file.getParent());
-        try (OutputStream out = Files.newOutputStream(file, StandardOpenOption.CREATE)) {
+        try (OutputStream out = Files.newOutputStream(file, StandardOpenOption.CREATE, StandardOpenOption.TRUNCATE_EXISTING)) {
```

Comment:
I would rather rely on the standard options of https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#newOutputStream(java.nio.file.Path,%20java.nio.file.OpenOption...)

> If no options are present then this method works as if the CREATE, TRUNCATE_EXISTING, and WRITE options are present

Reply:
Ah, indeed, thanks. Changed that. I'm wondering whether the alternative approach https://github.com/stoerr/jackrabbit-filevault/commit/916c7d0238435d8843ea506c95a87e70dc2bc6c8 would be a little better, though.